### PR TITLE
Concert

### PR DESCRIPTION
**Pull request by 29td**
Creating my first capstone project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+
+## Attribution note
+
+- I've add my linkedin ID to the readme file 
+- Removed the live demo link
+
+## Acknowledgments
+
+- Special thanks o all those who help me out with my code when i was strungling.
+- It is such a great honour to be sharing my work with everyone. I know some may learn something from my work some may help me learn new things too.
```
Comment (vechicin) [inline README.md]: Screen Shot 2022-06-30 at 8 15 48

Please fix it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,65 @@
...
+
+## Built With
+
+- HTML/CSS/JS
+- Node.js
+- git-bash/Vscode
+- laptop windows 10
+
+## Live Demo (if available)
+https://29td.github.io/My-First-Portfolio/
```
Comment (vechicin) [inline README.md]: You submitted a demo to your portfolio, please fix it.

Screen Shot 2022-06-30 at 8 17 14

File: about.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    </div>
+    </div>
+
+    <header>
+    <nav class="menu">
+        <a href="#link"><img src="./assets/Images/logo.png" alt="img"/></a>
+        <ul class="icons" id="nav-icons">
+            <li class="icon" >
+                <a href="about.html" class="icon-link">About</a>
+            </li>
```
Comment (vechicin) [inline about.html]: There should be a link that allows users to navigate back to the home page, please add it.

Screen Shot 2022-06-30 at 8 20 40

File: about.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+</head>
+<body>
+    <div class="network">
+        <div class="networks">
+            <a href="#"><i class="bi bi-facebook"></i></a>
+            <a href="#"><i class="bi bi-twitter"></i></a>
+            <a href="#" class="icon-lin">Logout</a>
+        <a href="#" class="icon-lin">Mypage</a>
+        <a href="#" class="icon-lin">Korean</a>
+    </div>
```
Comment (vechicin) [inline about.html:20]: This content should be aligned further right to match the template design, please fix it.

Additionally, you need to use the font provided in the template design, please fix that as well.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    </div>
+
+    <header>
+    <nav class="menu">
+        <a href="#link"><img src="./assets/Images/logo.png" alt="img"/></a>
```
Comment (vechicin) [inline about.html:25]: Please fix the logo's image size, it looks very squished right now.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+</section>
+<div id="featuredM">
+<h2>Featured Performers</h2>
+<div class="line">
+</div>                  
+</div>
+
+<footer id="footer">
+    <img src="./assets/Images/logo.png" alt="img"/><span>@copy right reserver summer festival 2022</span>
```
Comment (vechicin) [inline index.html:97]: This text is not visible, please fix its color.

You should fix the image size as well.

Screen Shot 2022-06-30 at 8 38 19

Please fix that.

Screen Shot 2022-06-30 at 8 38 38

Please add a footer to your `About` page and make sure the background colors are correct.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+    <img src="./assets/Icons/sunny.svg" alt="img" class="sunny"/>
+    <h2>Thursday/1 Sep from: 12:00pm-late</h2>
+    <h4>featuring Dj zinhle, Dladla Mshunqisi Qtwins ft Dj Tira Black Coffee and more</h4>
+    <div class="detail">
+        Get your tickets at shoprite or from R250
+        @Durban Beach
+        RSVP:<br>
+        <a href = "tel:076344711">+27 76 344 711</a>,
+        <a href = "malito:nonhlanhlamndebele9@gmail.com">nonhlanhlamndebele9@gmail.com</a>
+    </div>
```
Comment (vechicin) [inline index.html]: Screen Shot 2022-06-30 at 8 44 04

Please make sure this section fulfills all of these requirements.

Comment (vechicin) [inline index.html]: This content is not centered, please fix it.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+        <li><img src="/assets/Images/beer.png" alt="img"/>Beer and gin garden<span><p>Drink and have fun</p></span></li>
+        <li><img src="/assets/Images/kids.png" alt="img"/>Kids' entertainment<span><p>during the day of the event the kids can have fun as well.</p></span></li>
+        <li><img src="/assets/Images/cooler.png" alt="img"/>Cooler box allowed from 12pm-5pm<span><p>from this time you dont have to pay for your cooler-box</p></span></li>
+        <li><img src="/assets/Images/cooler.png" alt="img"/>From 6pm-late cooler box fee R100<span><p>from this time you have to pay for your cooler-box</p></span></li>
+        <li><img src="/assets/Images/refund.png" alt="img"/>No Refunds<span><p>No refunds for every payment you make.</p></span></li>
+    </ol>
+    <button  class="global" type="submit">Join Global Link</button>
+    <button  class="global1" type="submit">SEE THE WHOLE PROGRAM</button>
+
+</div>
```
Comment (vechicin) [inline index.html]: The content is not centered, please fix it.

Additionally, in the mobile version, the margins are not quite right and some of the content is overflowing into the container below, please fix it.

Screen Shot 2022-06-30 at 8 51 51

Please fix this issue for the mobile version as well.

Screen Shot 2022-06-30 at 8 53 03

Fix this issue as well.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+</section>
+<div id="featuredM">
+<h2>Featured Performers</h2>
+<div class="line">
+</div>                  
+</div>
```
Comment (vechicin) [inline index.html]: The content is not centered, please fix it.

Screen Shot 2022-06-30 at 8 48 51

Make sure this section fulfills all of these requirements.

File: about.html [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+                </div>
+                <div class="us">
+                    <img src="./assets/Images/fest.png" alt="img" class="img"/>
+                    <h1>2017</h1>
+                    <p>Summer festival 2017 in CapeTown</p>
+                </div>
+        
+            </div>
+        </div>
+</section>
```
Comment (vechicin) [inline about.html]: Screen Shot 2022-06-30 at 8 58 12

Your `About` page design is very disorganized. Please make sure you center all of your content properly and make sure the text over the images is clearly visible.

File: script.js [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+  featuredImage: './assets/Images/mc.png',
+  linkToLiveVersion: 'https://29td.github.io/concert-website/',
+  linkToSource: 'https://github.com/29td/concert-website/tree/concert',
+}, {
+  name: 'Tatiana',
+  description: 'Kenya based musician.',
+  featuredImage: './assets/Images/mc.png',
+  linkToLiveVersion: 'https://29td.github.io/concert-website/',
+  linkToSource: 'https://github.com/29td/concert-website/tree/concert',
+}];
```
Comment (vechicin) [inline script.js:65]: Screen Shot 2022-06-30 at 9 04 23

Please make sure your speaker objects hold all of the information above.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @29td ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️

- [x] Requesting your project to be seen at a specific zoom percentage is not acceptable because your users won't know they need to do this. You need to make sure your content works at 100% zoom, please fix it.

- [x] Please change your PR title/description to something more descriptive

- [x] You need to submit a video presentation for this project, please do so.
 
Screen Shot 2022-06-30 at 8 14 47

- [x] You need to match the template design, please fulfill all the requirements below.

Screen Shot 2022-06-30 at 9 00 49

- [ ] The way your HTML code is structured is not correct, please fulfill all the requirements below.

Screen Shot 2022-06-30 at 9 02 11


_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @29td     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**